crates/vm/levm/src/opcode_handlers/system.rs

```diff
@@ -94,12 +94,13 @@ impl VM {
     }
 
     // CALLCODE operation
+    // TODO: Add gas consumption
     pub fn op_callcode(
         &mut self,
         current_call_frame: &mut CallFrame,
     ) -> Result<OpcodeSuccess, VMError> {
         let gas = current_call_frame.stack.pop()?;
-        let code_address = Address::from_low_u64_be(current_call_frame.stack.pop()?.low_u64());
+        let code_address = word_to_address(current_call_frame.stack.pop()?);
         let value = current_call_frame.stack.pop()?;
         let args_offset = current_call_frame.stack.pop()?.try_into().unwrap();
```

Comment:
Maybe out of the scope of this PR, but maybe we can also handle these unwraps below.